pkg/services/serviceaccounts/database/store.go

```diff
@@ -318,7 +318,7 @@ func (s *ServiceAccountsStoreImpl) SearchOrgServiceAccounts(ctx context.Context,
 				whereConditions,
 				"is_disabled = ?")
 			whereParams = append(whereParams, s.sqlStore.GetDialect().BooleanStr(true))
-		case serviceaccounts.FilterOnlyExternal:
+		case serviceaccounts.FilterOnlyManaged:
```

Comment:
Why did you rename the filter?
I was thinking we should consider external and managed to be two different concepts. 
As of today external service accounts are the only managed service accounts, but that might not be true tomorrow.